Update 20191119.md (#6)
p2pcommons/working-group: 20191119.md

-   [`@p2pcommons/specs`](https://github.com/p2pcommons/specs)
    - Update by Chris
      - Writing specification
        - HTML with a linter to make sure e.g., Markdown

>           - Julian: Good to test iwth various things, e.g., with pandoc and we could use Docker

Reviewer: ```suggestion
          - Julian: Good to test with various things, e.g., with pandoc and we could use Docker
```

>       - What specs do we need in line with the vision?

Reviewer 2: ```suggestion
      - What specs do we need to be in line with the vision?
```

        - Using a super new module from the Hyperdrive v10 version: [`corestore-swarm-networking`](https://github.com/andrewosh/corestore-swarm-networking) --> need to debug more, but we're close to getting a working version
    - Questions
      - Can we do version specific clones or do they need to be complete clones? [open question]
        - Chris asks because of the bandwidth issue for creating the registries and post hoc updates to modules.
        - Diego: Now we're cloning the key, so the full one. Unsure right now, we will figure out when we're at checkout version implementation and in version cloning.
-   [`@hypergraph-xyz/cli`](https://github.com/hypergraph-xyz/cli)
    - Update by Julian
      - Started with integrating latest version of `sdk-js`, but waiting on the new one that has the flattened object.
    - Questions
      - Chris: Had some issues
        - Julian: Let's do a major dogfooding sessions at `v0.3.0`
-   [`@hypergraph-xyz/desktop`](https://github.com/hypergraph-xyz/desktop)
    - Update by Julian
      - Set up Electron
      - With tests included :)

>       - Next include code-reloading and built infrastructure (with release assets)

Reviewer 2: ```suggestion
      - Next include react, code-reloading and build infrastructure (with release assets)
```

      - When that's done, we can play around with wax editor
        - The [wax-v1 editor](https://wax-demo.coko.foundation) has rough edges but seems something we can work with
        - Easy to swap out editors
      - CHJH: Important to get the writing/text spec for this --- what kind of HTML do we want?

>         - Julian: What's allowed? We can write a linter that removes parts. 

Reviewer 2: ```suggestion
        - JG: What's allowed? We can write a linter that removes parts. 
```

        - JG: We're in a difficult position, markdown is safe but html not much so. Sandboxing might help
        - CHJH: We might take example from iodide, jupyter notebooks, [observable HQ](https://observablehq.com/)
    - Questions
      - How to plan for scalable localization?
        - JG: Not so difficult for the interface. We might look at nodejs. For example [CrowdIn](https://crowdin.com/)
        - JG: We could also do translation! Add a subtype. How to do this from a UX perspective? 
-   `@hypergraph-xyz/mobile`
    - For later reference:
      - Apple [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/)
      - Google [Play Store Launch Checklist](https://developer.android.com/distribute/best-practices/launch/launch-checklist)
    - Questions
      - Hypergraph Authenticator
        - Hyperdrive `v10` is backed by `hypertrie`; to what extent is this usable right now?

>           - DP: Hypertrie is stable, being used. Mathias and Andrew Osh are the main people working on this, primarily working on performance. Performance not an issue for us now, so we good to go on that. To use, we'd need to integrate `kappa`. A colleague at Geut could help maybe. They're talking about `party` and `feats`, party authorises.

Reviewer: ```suggestion
          - DP: Hypertrie is stable, being used. Mathias and Andrew are the main people working on this, primarily working on performance. Performance not an issue for us now, so we good to go on that. To use, we'd need to integrate `kappa`. A colleague at GEUT could help maybe.
```
